requirements/requirements.in

```diff
@@ -105,6 +105,7 @@ csv342==1.0.0
 Faker==0.8.15
 polib==1.1.0
 werkzeug==0.11.15
+WeasyPrint==0.39
```

Comment:
Why is this using version 0.39? Quickly going over weasyprint's change log, it should be possible to use 0.42.3 with python 2.

Reply:
You are right, I had to misinterpret the support part. I have tested it, 0.42.3 works as well, I am adding the change.

Reply:
I think you still need to add it to `uninstall-requirements.txt` as emord said